lua/bufferline/utils.lua

```diff
@@ -15,9 +16,11 @@ local set_hl = vim.api.nvim_set_hl
 --- @return integer|string color
 local function attribute_or_default(groups, index, default, guicolors)
   for _, group in ipairs(groups) do
-    local hl = get_hl_by_name(group, guicolors)
-    if hl[index] then
-      return guicolors and ('#%06x'):format(hl[index]) or hl[index]
+    if hlexists(group) > 0 then
```

Comment:
A bugfix. I stumbled upon this by referencing missing default highlight groups for `fg_hint` and `fg_info` in the highlights module (`HintMsg` and `InfoMsg` respectively). Users may have manually cleared some of the default highlight groups (i.e. the ones we rely on for `highlight.setup`), and that would cause an error.